src/Sulu/Bundle/WebsiteBundle/Twig/Seo/SeoTwigExtension.php

```diff
@@ -78,7 +78,7 @@ public function getFunctions()
     public function renderSeoTags(array $seoExtension, array $content, array $urls, $shadowBaseLocale)
     {
         $request = $this->requestStack->getCurrentRequest();
-        $requestSeo = $request->get('_seo', []);
+        $requestSeo = $request->attributes->get('_seo', []);
```

Comment:
👍 We should be more explicit about where to get our request data from in general.